src/ansys/openapi/common/_util.py

```diff
@@ -359,7 +370,9 @@ def from_dict(cls, configuration_dict: "RequestsConfiguration") -> "SessionConfi
         return new
 
 
-def handle_response(response: requests.Response) -> requests.Response:
+def handle_response(
+    response: requests.Response, response_model: DeserializedType = None
+) -> requests.Response:
```

Comment:
This function is kind of weird. It no longer needs to return anything, so really all it's doing is encapsulating the "if not 2xx then raise" logic.

Maybe we just delete the function and include the if-check in `__call_api`?

Reply:
I think so. As you say, it's not a private function, but it's also not documented. For me this falls into a grey area, and I'm happy to take the chance that either no one's using it, or if they are, they can easily work around this change.